source/includes/entityref/paybill/_clientcorporationrateagreementcard.md

```diff
@@ -0,0 +1,120 @@
+# Pay and Bill - ClientCorporationRateAgreementCard
+
+An effective-dated entity that represents all the rates on a Client Corporation.
+ClientCorporationRateAgreementCard is the flattened entity. When making PUT and POST calls
+to ClientCorporationRateAgreementCard, the child entities, ClientCorporationRateAgreementCardLineGroup and ClientCorporationRateAgreementCardLine,
+can be passed as well with all associated data.
+
+CRUD Access - CREATE, READ, UPDATE, DELETE
+
+<table>
+    <colgroup>
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+    </colgroup>
+    <thead>
+        <tr class="header">
+            <th>ClientCorporationRateAgreementCard fields</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Not null</th>
+            <th>Read-only</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr class="even">
+            <td>id</td>
+            <td>Integer</td>
+            <td>Unique identifier for this entity.</td>
+            <td>X</td>
+            <td>X</td>
+        </tr>
+        <tr class="odd">
+            <td>clientCorporation</td>
+            <td>To-one association</td>
+            <td>ClientCorporation</td>
+            <td>X</td>
+            <td>X</td>
+        </tr>
+        <tr class="even">
+            <td>clientCorporationRateAgreementCardLineGroups</td>
+            <td>To-many association</td>
+            <td>ClientCorporationRateAgreementCardLineGroup</td>
+            <td></td>
```

Comment:
Try to fill in this and all other missing descriptions on these files.
I realize things are missing from some existing PayBill entity pages but should not be. If the descriptions exist on other entity pages, copy from those to match.